frontend/src/components/generic/SocialLinks/SocialIcons.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faBriefcase, faEnvelope } from '@fortawesome/free-solid-svg-icons'
+import {
+    faGithub,
+    faLinkedin,
+    faDiscord,
+    faSlack,
+} from '@fortawesome/free-brands-svg-icons'
+import junctionStyle from 'utils/styles'
+
+const SocialIcons = ({ social, email = null }) => {
+    const classes = junctionStyle()
+
+    const socials = [
+        social.github && { icon: faGithub, link: social.github },
+        social.linkedin && { icon: faLinkedin, link: social.linkedin },
+        social.slack && { icon: faDiscord, link: social.slack },
+        social.discord && { icon: faSlack, link: social.discord },
+        social.portfolio && {
+            icon: faBriefcase,
+            link: social.portfolio,
+        },
+        email && {
+            icon: faEnvelope,
+            link: `https://mail.google.com/mail/?view=cm&to=${email}`,
```

Comment:
- [ ] Required: This section assumes all users will use gmail as their email client, which might not be the case for many, use a more broad approach like:
`onClick={() =>
popupCenter({
url:`mailto:${profile.email}`,
title:'email',
})}`

You can see an example on the file frontend/src/components/Participant/ProfileTop/index.js

https://github.com/wickathou/JunctionApp/blob/0241d55b031a869e0a5b8d8e7c7944bea9412833/frontend/src/components/Participant/ProfileTop/index.js#L246

Reply:
Thanks @wickathou, I will make the changes and request a new review.